handlers/deploy.go

```diff
@@ -296,6 +299,18 @@ func makeServiceSpec(request requests.CreateFunctionRequest) *v1.Service {
 	return serviceSpec
 }
 
+func buildAnnotations(request requests.CreateFunctionRequest) map[string]string {
```

Comment:
@LucasRoesler @alexellis I've added unit test to this method

Reply:
looks good